### PR TITLE
docs: improve readme file

### PR DESCRIPTION
**Pull request by dustytrinkets**
## What’s the focus of this PR
This PR is focused on updating the readme file so it's easier to get started with cuckoojs.
WIP please add anything you feel is needed 
Screenshot 2022-10-26 at 15 23 22

## How to review this PR
Think yourself as someone who doesn't know what cuckoo is or how to use it. What would you like to find here?

Before submitting this PR, I made sure:

[x] The code builds clean without any errors or warnings
[x] I'm using our guidelines
[ ] I've added unit tests if feasible

### CODE REVIEW

**Review by neodmy — APPROVED** [review]
👍

File: README.md [hunk 1]
```diff
@@ -3,31 +3,25 @@
 </div>
 <br />
 
-# Getting Started With Schematics
+#####Let the cuckoo accommodate the nest for you.
```
[the remark this answers is not in the document]
Comment (ardguezsoc) [reply README.md:6]: Screenshot 2022-10-27 at 10 02 50 AM
see how it is current displayed

**Review by ardguezsoc — APPROVED** [review]

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (dustytrinkets) [reply README.md]: thanks!

File: README.md [hunk 2]
```diff
@@ -3,31 +3,25 @@
 </div>
 <br />
 
-# Getting Started With Schematics
+##### Let the cuckoo accommodate the nest for you.
```
Comment (inigomarquinez) [inline README.md:6]: LOVING THIS! ❤️

File: README.md [hunk 3]
```diff
@@ -3,31 +3,25 @@
...
-To test locally, install `@angular-devkit/schematics-cli` globally and use the `schematics` command line tool. That tool acts the same as the `generate` command of the Angular CLI, but also has a debug mode.
 
-Check the documentation with
 
-```bash
-schematics --help
-```
-
-### Unit Testing
+### <img alt="logo" src="./assets/logo.svg" width="20px" />  How to use
```
Comment (inigomarquinez) [inline README.md:17]:
```suggestion
### <img alt="logo" src="./assets/logo.svg" width="20px" />  How to use it
```

File: README.md [hunk 4]
```diff
@@ -3,31 +3,25 @@
...
 
-`npm run test` will run the unit tests, using Jasmine as a runner and test framework.
 
-### Publishing
-
-To publish, simply do:
 
 ```bash
```
Comment (inigomarquinez) [inline README.md:21]: Perhaps we can add a description for the command, such as:

```suggestion
```bash
Creating a new NestJS app with the basic tooling:
```

File: README.md [hunk 5]
```diff
@@ -3,31 +3,25 @@
...
-npm run build
-npm publish
+cuckoo new <projectname>
 ```
 
-That's it!
+
+You can still use all the commands that work with NestJS
+
```
Comment (inigomarquinez) [inline README.md:27]: Perhaps we can add some examples:
```suggestion
cuckoo generate|g controller <name>
```

**Review by inigomarquinez — CHANGES_REQUESTED** [review]
Great work.

Some suggestions added :)

File: README.md (hunk 5 above)
Comment (dustytrinkets) [reply README.md:27]: nice!

**Pull request merged**